artiq/gateware/targets/kasli.py

```diff
@@ -462,6 +465,9 @@ def __init__(self, rtio_clk_freq=125e6, enable_sata=False, *, with_wrpll=False,
         disable_cdr_clk_ibuf = Signal(reset=1)
         disable_cdr_clk_ibuf.attr.add("no_retiming")
         if self.platform.hw_rev == "v2.0":
+            self.submodules.error_led = gpio.GPIOOut(Cat(
+                self.platform.request("error_led")))
+            self.csr_devices.append("error_led")
```

Comment:
Merged a bit too fast.

This LED code does not belong with the transceiver clocking code.